src/arkouda_server.chpl

```diff
@@ -311,11 +311,7 @@ proc main() {
                     var binaryRepMsg = segStrTondarrayMsg(cmd, args, st);
                     sendRepMsg(binaryRepMsg);
                   }
-                  when "lshdf"             {repTuple = lshdfMsg(cmd, args, st);}
-                  when "readAllHdf"        {repTuple = readAllHdfMsg(cmd, args, st);}
-                  when "tohdf"             {repTuple = tohdfMsg(cmd, args, st);}
-                  when "readAllParquet"    {repTuple = readAllParquetMsg(cmd, args, st);}
-                  when "writeParquet"      {repTuple = toparquetMsg(cmd, args, st);}
+
```

Comment:
These commands moved into their respective modules and get registered through the CommandMap to take advantage of our new modular build capability.